src/styles/until/_variables.scss

```diff
@@ -0,0 +1,18 @@
+$size-square:100px;
+$gap-square: 1px;
+$color-square:#eee;
+
+$max-width: 7 * $size-square + $gap-square * 7 + 10 * 2;
+
+$days:
+  "mon" ($size-square+$gap-square),
+  "tue" 2 * ($size-square+$gap-square),
+  "wed" 3 * ($size-square+$gap-square),
+  "thu" 4 * ($size-square+$gap-square),
+  "fri" 5 * ($size-square+$gap-square),
+  "sat" 6 * ($size-square+$gap-square);
```

Comment:
```suggestion
$days:
  "mon" ($size-square+$gap-square),
  "tue" 2 * ($size-square+$gap-square),
  "wed" 3 * ($size-square+$gap-square),
  "thu" 4 * ($size-square+$gap-square),
  "fri" 5 * ($size-square+$gap-square),
  "sat" 6 * ($size-square+$gap-square);
```
Monday should always start at the first column.
Hint:  you should change your code in this place for correct mapping 😉

Reply:
No. In American style Sunday it's first day in week.
![image](https://user-images.githubusercontent.com/108505176/180270632-83d1f6c6-6a98-4218-953d-f6e0fdd860dd.png)

Reply:
Ok, but in our task, it should start in the first line

![Screenshot 2022-07-21 at 20 43 01](https://user-images.githubusercontent.com/87187483/180278955-bf241ccb-1efe-4ae5-9d86-2abebb6668ab.png)

Reply:
i can approve, no problem)